dev/devicelab/lib/framework/ab.dart

```diff
@@ -207,12 +207,7 @@ class ABTest {
     return buffer.toString();
   }
 
-  Set<String> get _allScoreKeys {
-    return <String>{
-      ..._aResults.keys,
-      ..._bResults.keys,
-    };
-  }
+  Set<String> get _allScoreKeys => <String>{..._aResults.keys, ..._bResults.keys};
```

Comment:
Once we start using `dart format`, the right-hand-side will get moved to a new line, at which point it would violate [Consider using `=>` for short functions and methods](https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#consider-using--for-short-functions-and-methods):

> But only use `=>` when everything, including the function declaration, fits on a single line.

Technically, I'm lying—because that rule only applies to functions, not getters. But it would feel odd to have such a distinction. 

Consider

```dart
  Set<String> get _allScoreKeys => <String>{
        ..._aResults.keys,
        ..._bResults.keys,
      };
```

Reply:
> it would feel odd to have such a distinction.

I agree—there shouldn't be a line break between the getter name and where its value begins.

<br>

It's possible that this repo would use "tall style" (https://github.com/dart-lang/dart_style/issues/1253), which I believe would automatically put the items on their own lines.
And based on what I read in https://github.com/flutter/flutter/pull/153732, we also might set a >80 character line length.

<br>

But for now, since there's no way to know what it'll end up looking like, I think the approach you're showing is the best way to go 👍